Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -83,7 +83,7 @@ static inline void GenerateVertexShader(T& out, u32 components, API_TYPE api_typ
 		else
 		{
 			// Let's set up attributes
-			for (size_t i = 0; i < 8; ++i)
+			for (int i = 0; i < 8; ++i)
```

Comment:
This should be a u32 (the same type as numTexGens), printf specifier will just need to be changed to %u